app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesRecordGlobal.java

```diff
@@ -126,6 +128,8 @@ public String toString() {
 
     private LocalDate lastSentTrackingInfoDate = null;
 
+    private boolean alternatePasteBehavior = false;
```

Comment:
This should probably be  `true`? At least until there is a fix for it in OpenJFX, and we have a release that we can use.

Reply:
Good point, I'll add a platform specific default. Shall there be any constraints or shall it be true for MacOS in any case? (false for other OS).

Reply:
@Oliver-Loeffler You might have missed my question above?

Reply:
Hi @jperedadnr , well, I'll rework this, I understand your point and agree. But currently I am lacking the concentration to do this, my mind is blocked by all the stupidity which is ongoing currently - this makes me really sad and no, don't wanna code, it does not help my mind.

Reply:
No worries.
Respect!

Reply:
Hi all, I'll conclude this work until end of this week. Sorry for the delay on that.

Reply:
Hi @Oliver-Loeffler , if you are still up for it, it would be great if you could finish this PR. We are about to do a RC with Java/JavaFX 19, and it could probably make it in time for the SB 19 release. Thank you!

Reply:
Hi @jperedadnr, yes I am. Will clean up/refactor accordingly. Hope to get it ready tomorrow.

Reply:
Now the alternate paste behavior is active by default on macOS. Thank you for pinging.